scripts/python/gen.py

```diff
@@ -244,38 +231,18 @@ def _create_inventory(self):
         log = logger.getlogger()
 
         cont = Container(self.config_file_path, self.args.create_inventory)
+        cont.copy(self.config_file_path, self.cont_config_file_path)
         cmd = []
         cmd.append(gen.get_container_venv_python_exe())
         cmd.append(os.path.join(
             gen.get_container_python_path(), 'inv_create.py'))
         cmd.append(self.cont_config_file_path)
         try:
-            cont.run_command(cmd)
+            cont.run_command(cmd, interactive=True)
         except UserException as exc:
             print('Fail:', str(exc), file=sys.stderr)
             sys.exit(1)
 
```

Comment:
Is the inventory.yml file still resident in the container and sym-linked in the power-up directory?

Reply:
The inventory now resides in the host power-up dir and is mounted inside the container (as a bind mount: https://docs.docker.com/storage/bind-mounts/):
https://github.com/open-power-ref-design-toolkit/power-up/pull/298/files#diff-fde31366ffd05c631262381c65ee9d84R123

This allows read/write access inside and outside of the container. I'm not sure if that's a good or bad thing.